windowspkg/linkerd2-cli/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,14 @@
+﻿$ErrorActionPreference = 'Stop';
+$toolsPath = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+
+$packageArgs = @{
+
+  packageName    = 'linkerd'
+  fileFullPath   = "$toolsPath\linkerd.exe"
+  url            = 'https://github.com/linkerd/linkerd2/releases/download/stable-2.6.1/linkerd2-cli-stable-2.6.1-windows.exe'
+  checksum       = '4ed915b10c2a76070d40bf3223f09e742777105914acaeadfc83d99ef6cf1798'
+  checksumType   = 'sha256'
+}
+
+Get-ChocolateyWebFile @packageArgs
+Install-ChocolateyPath $packageArgs.fileFullPath 'Machine'
```

Comment:
IIUC, this adds the `tools\linkerd.exe` path to the global `$PATH`, right? Can we use the `User` path type so that the script doesn't require UAC/admin privileges on the machine?

Reply:
Ah! that's a good catch. Thanks for pointing that out :) Will look into and modify the script to allow for use without admin privileges.

Reply:
I believe if you use the `Install-Chocolatey*` family of commands, ShimGen should make a shim in the ProgramData/chocolatey/bin folder automatically, hence not requiring a modification of the path variable. It's for example how Helm does it:

```
$packageName = 'kubernetes-helm'
$toolsDir =  "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
$tempDir = "$toolsDir\temp"

$packageArgs = @{
    PackageName    = $packageName
    Url64bit       = 'https://get.helm.sh/helm-v3.1.1-windows-amd64.zip'
    Checksum64     = '5efc1c7bdea42a17a1ea197a3a2e47c8f1b0a5464894b98ae73a4c0620163040'
    ChecksumType64 = 'sha256'
    UnzipLocation  = $toolsDir
}

# Download and unzip into a temp folder
Install-ChocolateyZipPackage @packageArgs
```

That zip is a file with an `windows-amd64` folder in it with `helm.exe`, `LICENCE` and `README.md` in it. Rather straightforward.

Reply:
Ah! That's a interesting way to do it. Although currently Linkerd doesn't make a zip folder for the windows package. Will look into this to find analogous ways for how it currently is (if any). Thank you for suggesting this :)